src/scenes/RedirectsRouter/index.tsx

```diff
@@ -1,41 +1,23 @@
 import React from 'react';
-import { Redirect, Route, Switch } from 'react-router-dom';
+import { Navigate, Route, Routes } from 'react-router-dom';
 
 import { APP_PATHS } from 'src/constants';
 
 const RedirectsRouter = () => {
   return (
-    <Switch>
+    <Routes>
       {/* Redirects. Invalid paths will redirect to the closest valid path. */}
       {/* In alphabetical order for easy reference with APP_PATHS, except for /home which must go last */}
-      <Route path={APP_PATHS.ACCESSIONS + '/'}>
-        <Redirect to={APP_PATHS.ACCESSIONS} />
-      </Route>
-      <Route path={APP_PATHS.CHECKIN + '/'}>
-        <Redirect to={APP_PATHS.CHECKIN} />
-      </Route>
-      <Route path={APP_PATHS.CONTACT_US + '/'}>
-        <Redirect to={APP_PATHS.CONTACT_US} />
-      </Route>
-      <Route exact path={APP_PATHS.ORGANIZATION + '/'}>
-        <Redirect to={APP_PATHS.ORGANIZATION} />
-      </Route>
-      <Route exact path={APP_PATHS.PEOPLE + '/'}>
-        <Redirect to={APP_PATHS.PEOPLE} />
-      </Route>
-      <Route exact path={APP_PATHS.PROJECTS + '/'}>
-        <Redirect to={APP_PATHS.PROJECTS} />
-      </Route>
-      <Route path={APP_PATHS.SEEDS_DASHBOARD + '/'}>
-        <Redirect to={APP_PATHS.SEEDS_DASHBOARD} />
-      </Route>
-      <Route path={APP_PATHS.SPECIES + '/'}>
-        <Redirect to={APP_PATHS.SPECIES} />
-      </Route>
-      <Route path='*'>
-        <Redirect to={APP_PATHS.HOME} />
-      </Route>
-    </Switch>
+      {/* <Route path={APP_PATHS.ACCESSIONS + '/*'} element={<Navigate to={APP_PATHS.ACCESSIONS} />} /> */}
```

Comment:
why commented out?

Reply:
We don't need this redirect router anymore. I commented it to test, and looks like it's not necessary, because routes now behaves like we wanted, the '/*' is inside each sub-router